services/app-web/src/pages/CMSDashboard/RateReviewsDashboard/RateReviewsTable.tsx

```diff
@@ -80,6 +80,21 @@ const rateTypeOptions = [
     },
 ]
 
+const rateStatusOptions = [
+    {
+        label: 'Submitted',
+        value: 'SUBMITTED',
+    },
+    {
+        label: 'Unlocked',
+        value: 'UNLOCKED',
+    },
+    {
+        label: 'Withdrawn',
+        value: 'WITHDRAWN',
```

Comment:
![Screenshot 2025-01-08 at 6 24 53 PM](https://github.com/user-attachments/assets/530d6ceb-3f33-44e3-8488-6a8aeaaa634c)

Issue: On the submission dashboard, the filters are in title case so we should keep with that for the rate dashboard.